modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/IgniteStatisticsHelper.java

```diff
@@ -96,7 +92,7 @@ public ObjectStatisticsImpl aggregateLocalStatistics(
                         keyMsg.schema(), keyMsg.obj()));
         }
 
-        return aggregateLocalStatistics(tbl, filterColumns(tbl.getColumns(), keyMsg.colNames()), stats);
+        return aggregateLocalStatistics(tbl, filterColumns(tbl.getColumns(), keyMsg.colNames()), stats, log);
```

Comment:
aggreagateLocalStatistics doesn't use log and moreover it can obtain it from the same IgniteStatisticsHelper object.